### DISCUSSION TITLE
Nerd Font

### DISCUSSION THREAD

**Discussion by p145085** [erikw/tmux-powerline#446] [Q&A] [answered]
I installed (all) the fonts on my Windows host and also on my Ubuntu (WSL) but tmux-powerline still yields (?) symbols. I'm missing something but I don't know what.

**Comment by xx4h** [comment, marked as answer]
Did you only install them or also set them as the used font in WSL?

**Reply by p145085** [reply]
Welp, that was it. Thanks.
